operator/config/manifests/bases/rhacs-operator.clusterserviceversion.yaml

```diff
@@ -563,12 +563,10 @@ spec:
       - description: Custom environment variables to set on managed pods' containers.
         displayName: Environment Variables
         path: customize.envVars
-      - description: To ensure comprehensive monitoring of your cluster activity,
-          Red Hat Advanced Cluster Security will run services on every node in the
-          cluster, including tainted nodes by default. If you do not want this behavior,
-          please select 'AvoidTaints' here.
-        displayName: Taint Toleration
-        path: perNode.taintToleration
+      - description: Settings for the Node-Scanner container, which is responsible
+          for scanning the Nodes' filesystem
```

Comment:
```suggestion
          for scanning the Nodes' filesystem.
```

Reply:
Got that covered [0296560](https://github.com/stackrox/stackrox/pull/6160/commits/0296560e291e5a758622ace57dcc4d7ec9ab00ba)